client/utils/__tests__/form-spec.js

```diff
@@ -59,7 +62,9 @@ describe('formApiAdapter', () => {
       it('rejects with api error on failure', () => {
         const adapter = formApiAdapter(dispatch, actionCreator)
 
-        return expect(adapter(formValues)).to.be.rejected
+        return adapter(formValues).catch(failure => {
```

Comment:
I think this will silently pass if the promise is rejected.  Can use async/await and do something like:

`expect(async () => await adapter(formValues)).toThrow()` ?